operations/model.go

```diff
@@ -190,7 +191,7 @@ func (s *ClientSettings) SetDefaultVariants(project string, variants ...string)
 		}
 	}
 
-	s.Projects = append(s.Projects, ClientProjectConf{project, true, variants, nil})
+	s.Projects = append(s.Projects, ClientProjectConf{project, true, "", variants, nil})
```

Comment:
should probably change the ClienProjectConf constructor to use keyed field names